react/features/toolbar/defaultToolbarButtons.js

```diff
@@ -0,0 +1,212 @@
+ */
+export default {
+    'microphone': {
+        id: 'toolbar_button_mute',
+        tooltipKey: 'toolbar.mute',
+        classNames: [ 'button', 'icon-microphone' ],
+        enabled: true,
+        shortcut: 'M',
+        shortcutAttr: 'mutePopover',
+        shortcutFunc() {
+            JitsiMeetJS.analytics.sendEvent('shortcut.audiomute.toggled');
+            APP.conference.toggleAudioMuted();
+        },
+        shortcutDescription: 'keyboardShortcuts.mute',
+        popups: [
+            {
+                id: 'micMutedPopup',
+                className: 'loginmenu',
+                dataAttr: 'toolbar.micMutedPopup'
+            },
+            {
+                id: 'unableToUnmutePopup',
+                className: 'loginmenu',
+                dataAttr: 'toolbar.unableToUnmutePopup'
+            },
+            {
+                id: 'talkWhileMutedPopup',
+                className: 'loginmenu',
+                dataAttr: 'toolbar.talkWhileMutedPopup'
+            }
+        ],
+        content: 'Mute / Unmute',
+        i18n: 'toolbar.mute'
+    },
+    'camera': {
+        id: 'toolbar_button_camera',
+        tooltipKey: 'toolbar.videomute',
+        classNames: [ 'button', 'icon-camera' ],
+        enabled: true,
+        shortcut: 'V',
+        shortcutAttr: 'toggleVideoPopover',
+        shortcutFunc() {
+            JitsiMeetJS.analytics.sendEvent('shortcut.videomute.toggled');
+            APP.conference.toggleVideoMuted();
+        },
+        shortcutDescription: 'keyboardShortcuts.videoMute',
+        content: 'Start / stop camera',
+        i18n: 'toolbar.videomute'
+    },
+    'desktop': {
+        id: 'toolbar_button_desktopsharing',
+        tooltipKey: 'toolbar.sharescreen',
+        classNames: [ 'button', 'icon-share-desktop' ],
+        enabled: true,
+        shortcut: 'D',
+        shortcutAttr: 'toggleDesktopSharingPopover',
+        shortcutFunc() {
+            JitsiMeetJS.analytics.sendEvent('shortcut.screen.toggled');
+            APP.conference.toggleScreenSharing();
+        },
+        shortcutDescription: 'keyboardShortcuts.toggleScreensharing',
+        content: 'Share screen',
+        i18n: 'toolbar.sharescreen'
+    },
+    'invite': {
+        id: 'toolbar_button_link',
+        tooltipKey: 'toolbar.invite',
+        classNames: [ 'button', 'icon-link' ],
+        content: 'Invite others',
+        enabled: true,
+        i18n: 'toolbar.invite'
+    },
+    'chat': {
+        id: 'toolbar_button_chat',
+        tooltipKey: 'toolbar.chat',
+        classNames: [ 'button', 'icon-chat' ],
+        enabled: true,
+        shortcut: 'C',
+        shortcutAttr: 'toggleChatPopover',
+        shortcutFunc() {
+            JitsiMeetJS.analytics.sendEvent('shortcut.chat.toggled');
+            APP.UI.toggleChat();
+        },
+        shortcutDescription: 'keyboardShortcuts.toggleChat',
+        sideContainerId: 'chat_container',
+        html: <span className = 'badge-round'>
```

Comment:
This looks like it doesn't belong here. I understand you want to group all stuff related to a button somewhere, but this looks weird here.